### DISCUSSION TITLE
Solar-Passthrough confusion

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1125] [Q&A]
Posting screenshots as well for clarity.
Still I don’t really understand the principle.

Why if MPPT generates 106W and house consumption currently is 53W (so less than generated energy) - the inverter is still turned off? Isn’t it defying the purpose of Solar-Pathrough? 

![IMG_6896](https://github.com/user-attachments/assets/cb02fdd9-603d-4f49-847c-427471addacd)

![IMG_6897](https://github.com/user-attachments/assets/956f6f8c-f549-4e26-9d03-a91dced558f0)

**Comment by spcqike** [comment]
What is your DPL minimal power limit?

**Comment by greymda** [comment]
24W

<img width="1075" alt="image" src="https://github.com/user-attachments/assets/efd3d3fc-8c20-4b07-b93c-fe7a915644fa">

**Comment by spcqike** [comment]
Can you get any relevant logs with DPL verbose logging active?

**Comment by greymda** [comment]
@spcqike after some mumbo-jumbo it started, so in the console now not relevant logs. is there any option to download the full log file?

**Reply by spcqike** [reply]
no.

just keep an eye on it. if it happens again, like tomorrow, please have a look at the console.

**Reply by Manos1966** [reply]
> is there any option to download the full log file?

https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/How-to-Search-for-Solutions%2C-Ask-Questions%2C-Report-Problems#41-create-a-logfile-datei-with-console


And perhaps, deactivate option "Use Battery at night even if partially charged" ?
